modules/tinymce/src/core/test/ts/browser/dom/ContentCssCorsTest.ts

```diff
@@ -11,20 +12,118 @@ describe('browser.tinymce.core.dom.ContentCssCorsTest', () => {
     toolbar: false
   };
 
-  const assertCorsLinkPresence = (editor: Editor, expected: boolean) => {
+  const inlineSettings = {
+    ...settings,
+    inline: true
+  };
+
+  const assertCorsLinkPresence = (editor: Editor, expected: number) => {
+    // Classic mode has two stylesheets in the document head, content.css and content.min.css.
+    // Inline mode shares stylesheets with the outer document, and should have two different stylesheets by default, skin.min.css and content.inline.min.css.
     const corsLinks = editor.getDoc().querySelectorAll('link[crossorigin="anonymous"]');
-    assert.equal(corsLinks.length > 0, expected, 'should have link with crossorigin="anonymous"');
+    assert.equal(corsLinks.length, expected, `should have ${expected} link(s) with crossorigin="anonymous"`);
   };
 
-  it('assert crossorigin link presence with setting set', async () => {
-    const editor = await McEditor.pFromSettings<Editor>({ ...settings, content_css_cors: true });
-    assertCorsLinkPresence(editor, true);
-    McEditor.remove(editor);
+  afterEach(() => {
```

Comment:
Does this have to be `afterEach` or can it be `after`?

Reply:
`afterEach` since the global `styleSheetLoader` is affected each time `content_css_cors` is set in an inline editor. I wasn't able to find another way to reload `tinymce` in each test, so this was the next best thing.